packages/components/src/components/input/input.stories.ts

```diff
@@ -69,731 +35,219 @@ export const Default = {
 };
 
 /**
- * Use the `label` attribute to give the input an accessible label. For labels that contain HTML, use the `label` slot instead.
+ * Use the `size` attribute to change the size.
  */
 
-export const Labels = {
-  args: overrideArgs([{ type: 'attribute', name: 'label', value: 'Label' }]),
-  render: (args: any) => {
-    return html`
-      <div class="w-[250px]">
-        ${generateTemplate({
-          args
-        })}
-      </div>
-    `;
-  }
+export const Size = {
+  render: () => html`
+    <div class="grid grid-cols-2 gap-12 content-end">
+      <sd-input size="lg" label="Label"> </sd-input>
+      <sd-input size="md" label="Label"> </sd-input>
+      <sd-input size="sm" label="Label"> </sd-input>
```

Comment:
Please fill the placeholders from Figma:
![image](https://github.com/user-attachments/assets/1fe5880e-c6d6-4f3b-9185-bffa5cd99eb8)